webgpu.h

```diff
@@ -1524,6 +1621,10 @@ extern "C" {
 #if !defined(WGPU_SKIP_PROCS)
 
 typedef WGPUInstance (*WGPUProcCreateInstance)(WGPU_NULLABLE WGPUInstanceDescriptor const * descriptor) WGPU_FUNCTION_ATTRIBUTE;
+/**
+ * Query the supported instance features
+ */
```

Comment:
This is a separate issue but it would be nice to use `@copydoc` for the proc pointers, probably. https://github.com/webgpu-native/webgpu-headers/issues/315